src/main/java/seedu/address/model/meeting/UniqueMeetingList.java

```diff
@@ -92,7 +92,7 @@ public void removeMeetingsWithContact(Person contact) {
     public void updateMeetingsWithContact(Person contact) {
         requireNonNull(contact);
         List<Meeting> meetingsToUpdate =
-                internalList.stream().filter(meeting -> meeting.getPersonId() == contact.getId())
+                internalList.stream().filter(meeting -> meeting.getPerson().hasSameId(contact))
```

Comment:
Just checking, should `UniqueMeetingList#removeMeetingsWithContact` be updated as well? Likewise for `UniqueSaleList` and `UniqueReminderList`

Reply:
Hmm, I feel like it technically isn't needed compared to the updateModelWithContact methods, since removeModelsWithContact should provide the exact contact that matches the one in the model.

Reply:
I see. Thanks for clarifying! I'll leave Jin Ming to do the approving!